docs/making-your-own.md

```diff
@@ -24,33 +24,80 @@ However, people like to tinker, so instead of [layering packages](https://docs.f
-     - Add the rpm's you'd like to be included in your image
-     - Flatpaks are a work in progress but you'll be able to edit a file
-1. Generate a keypair
-   - Install the [cosign CLI tool](https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-install-cosign/)
-   - Run `cosign generate-key-pair`
-   - In your repository settings, under Secrets and Variables -> Actions
-     - Create a new secret, make sure that you don't put a password on the key as this will be used by an action:
-     ![image](https://user-images.githubusercontent.com/1264109/216735595-0ecf1b66-b9ee-439e-87d7-c8cc43c2110a.png)
-     - Call it `SIGNING_SECRET` and then paste the contents of `cosign.key` into the field and save it. Be careful to make sure it's the .key file and not the .pub file. It should look like this:  
-     ![image](https://user-images.githubusercontent.com/1264109/216735690-2d19271f-cee2-45ac-a039-23e6a4c16b34.png)
-     - Copy the `cosign.pub` key into the root of your repository, replacing the key you got from here.
-     - Copy the instructions from the verification section of this readme and make adjustments to your container url. This part is important, users must have a method of verifying the image. The linux desktop must not lag behind in cloud when it comes to supply chain security, so we're starting right from the start! (Seriously don't skip this part) 
-1. Start making modifications to your Containerfile!
-   - Change a few things and keep an eye on your Actions and Packages section of your repo, you'll generate a new image one every merge and additionally every day. 
-   - Follow the instructions at the top of this repo but this time with the `ghcr.io/yourusername/beagles` url and then you'll be good to go!
-   - Hang out in the [discussions forums](https://github.com/orgs/ublue-os/discussions) with others to share tips and get help, enjoy!
+
+### Set up container signing
+
+Container signing is important for end-user security and is enabled on all uBlue images. It is highly recommended you set this up, and by default the image builds *will fail* if you don't.
+
+This part is important, users must have a method of verifying the image. The Linux desktop must not lag behind in cloud when it comes to supply chain security, so we're starting right from the start! (Seriously don't skip this part) 
+
+!!! warning
+
+    Be careful to *never* accidentally commit `cosign.key` into your git repo.
+
+1. Generate a key pair
+    1. Install the [cosign CLI tool](https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-install-cosign/)
+    1. Run `cosign generate-key-pair` inside your repo folder
+        - Do NOT put in a password when it asks you to, just press enter. The signing key will be used in GitHub Actions and will not work if it is encrypted. 
+    1. Add the private key to GitHub
+        - You can do this easily if you have the `github-cli` installed by running `gh secret set SIGNING_SECRET < cosign.key`
```

Comment:
```suggestion
        - If you have the `github-cli` installed by running `gh secret set SIGNING_SECRET < cosign.key`
```

Reply:
Addressed in [ff70345](https://github.com/ublue-os/website/pull/99/commits/ff703452ff40b68af8fbd69a01988d296a7e20d9) with minor changes.